### PR TITLE
made Makefile a little more Linux-friendly, added more requirements t…

### PR DESCRIPTION
**Pull request by guymatz**
…o dev requirements file

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,9 +38,23 @@ req:
...
+ifeq ($(PLATFORM), darwin)
 	which atlas-run-standalone || brew tap atlassian/tap && brew install atlassian/tap/atlassian-plugin-sdk
+else ifneq ($(wildcard /etc/debian_version),)
+	sudo sh -c 'echo "deb https://sdkrepo.atlassian.com/debian/ stable contrib" >/etc/apt/sources.list.d/atlassian_development.list'
+	sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B07804338C015B73
+	sudo apt-get install apt-transport-https
+	sudo apt-get update
+	sudo apt-get install atlassian-plugin-sdk
+else ifneq ($(wildcard /etc/redhat-release),)
+	curl https://marketplace.atlassian.com/download/plugins/atlassian-plugin-sdk-rpm/version/42380 -o /tmp/atlassian-plugin-sdk.noarch.rpm
```
Comment (dbaxa) [inline Makefile]: The specification destination location is not secure. Please create a temporary directory to save it under using `mktemp -d`

File: Makefile [hunk 2]
```diff
@@ -29,9 +38,23 @@ req:
 install: prepare
 	$(PREFIX)pip install .
 
+# https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project/install-the-atlassian-sdk-on-a-linux-or-mac-system#InstalltheAtlassianSDKonaLinuxorMacsystem-Homebrew
 install-sdk:
-	# https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project/install-the-atlassian-sdk-on-a-linux-or-mac-system#InstalltheAtlassianSDKonaLinuxorMacsystem-Homebrew
+ifeq ($(PLATFORM), darwin)
```
Comment (dbaxa) [inline Makefile]: IMHO we shouldn't assume that people are running as a user with sudo or root access - i they don't have root / sudo access we can fail out and point them at a location to find out how to install the sdk.

File: Makefile [hunk 3]
```diff
@@ -29,9 +38,23 @@ req:
...
 	which atlas-run-standalone || brew tap atlassian/tap && brew install atlassian/tap/atlassian-plugin-sdk
+else ifneq ($(wildcard /etc/debian_version),)
+	sudo sh -c 'echo "deb https://sdkrepo.atlassian.com/debian/ stable contrib" >/etc/apt/sources.list.d/atlassian_development.list'
```
Comment (dbaxa) [inline Makefile]: If this fails do we call back to informing the user that the sdk cannot be installed ? (for example in the case of a user not having sudo access)

File: Makefile (hunk 1 above)
Comment (guymatz) [reply Makefile]: Thanks for the comments!!  Yes, will do this  . . .

File: Makefile (hunk 3 above)
Comment (guymatz) [reply Makefile]: I'm doing a check to see if the user is root (sudo will also show them as root)

**Comment by guymatz** [comment]
@dbaxa I'm getting this error from travis:
error in setup command: Error parsing /home/travis/build/pycontribs/jira/setup.cfg: IOError: [Errno 2] No such file or directory: 'ChangeLog'

I don't think this has anything to do with my code . . .  any ideas?   Thanks!

**Comment by hdost** [comment]
@guymatz If you want to rebase of of develop the changelog issue should be fixed for Travis for the time being.

**Pull request closed without merging**